src/main/java/de/themoep/inventorygui/InventoryGui.java

```diff
@@ -1286,7 +1286,7 @@ private void onInventoryClick(InventoryClickEvent event) {
             if (event.getInventory().equals(getInventory(event.getWhoClicked()))) {
 
                 int slot = -1;
-                if (event.getRawSlot() < event.getView().getTopInventory().getSize()) {
+                if (event.getRawSlot() < GuiView.of(event.getView()).getTopInventory().getSize()) {
```

Comment:
Seeing as the reflection might thrown an exception I would prefer this being cancelled in the case of an error too.